sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -46,11 +46,21 @@ extensions:
         git log -1 --pretty=%h >> "${jobs_repo}/ORIGIN_COMMIT"
         ( source hack/lib/init.sh; os::build::rpm::get_nvra_vars; echo "-${OS_RPM_VERSION}-${OS_RPM_RELEASE}" ) >> "${jobs_repo}/ORIGIN_PKG_VERSION"
     - type: "script"
+      title: "copy openshift images from docker storage to CRI-O storage"
+      timeout: 10800
+      script: |-
+        for i in $(docker images --format '{{.Repository}}:{{.Tag}}' | grep "openshift\/origin"); do
+          sudo skopeo copy docker-daemon:$i containers-storage:\[overlay2@/var/lib/containers/storage+/var/run/containers/storage:overlay2.override_kernel_check=1\]$i
+        done
+    - type: "script"
       title: "install origin"
       timeout: 7200
       repository: "aos-cd-jobs"
       script: |-
         local_ip="$( curl http://169.254.169.254/latest/meta-data/local-ipv4 )"
+        # debug for Dan
+        sudo setenforce 0
```

Comment:
I'll remove this in a follow up, need to test separately what's going on /cc @mrunalp

Reply:
Please open an issue to track this.